data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/CracCleaner.java

```diff
@@ -142,6 +143,24 @@ public CracCleaner() {
         removedStates.forEach(state -> crac.removeState(state.getId()));
         report.forEach(LOGGER::warn);
 
+        // remove PstRanges with a RELATIVE_TO_PREVIOUS_INSTANT for preventive PST RAs
```

Comment:
maybe add a unit test on this behaviour

Reply:
added a test for this in the testCleanCrac unit test